net-misc/tigervnc/tigervnc-1.10.0.ebuild

```diff
@@ -0,0 +1,197 @@
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sh ~sparc ~x86"
+IUSE="dri3 +drm gnutls java libressl nls +opengl pam server xinerama +xorgmodule"
+
+COMMON_DEPEND="
+	>=x11-libs/fltk-1.3.1
+	sys-libs/zlib:=
+	virtual/jpeg:0
+	x11-libs/libX11
+	x11-libs/libXext
+	x11-libs/libXrender
+	gnutls? ( net-libs/gnutls:= )
+	nls? ( virtual/libiconv )
+	pam? ( sys-libs/pam )
+	xinerama? ( x11-libs/libXinerama )
+	server? (
+		x11-apps/xsetroot
+		x11-libs/libXau
+		x11-libs/libXdamage
+		x11-libs/libXdmcp
+		x11-libs/libXfont2
+		x11-libs/libXtst
+		>=x11-apps/xauth-1.0.3
+		>=x11-libs/pixman-0.27.2
+		>=x11-misc/xkeyboard-config-2.4.1-r3
+		drm? ( x11-libs/libdrm )
+		!libressl? ( dev-libs/openssl:0= )
+		libressl? ( dev-libs/libressl:0= )
+		xorgmodule? ( =x11-base/xorg-server-${XSERVER_VERSION%.*}* )
+	)
+	"
+
+RDEPEND="${COMMON_DEPEND}
+	!net-misc/tightvnc
+	!net-misc/vnc
+	!net-misc/xf4vnc
+	java? ( >=virtual/jre-1.5:* )"
+
+DEPEND="${COMMON_DEPEND}
+	media-libs/fontconfig
+	x11-base/xorg-proto
+	x11-libs/libICE
+	x11-libs/libSM
+	x11-libs/libXcursor
+	x11-libs/libXfixes
+	x11-libs/libXft
+	x11-libs/libXi
+	java? ( >=virtual/jdk-1.5 )
+	nls? ( sys-devel/gettext )
+	server? (
+		>=x11-libs/xtrans-1.3.3
+		dev-libs/libbsd
+		media-fonts/font-util
+		virtual/pkgconfig
+		x11-libs/libxkbfile
+		x11-libs/libxshmfence
+		x11-misc/util-macros
+		opengl? ( >=media-libs/mesa-10.3.4-r1 )
+	)"
+
+PATCHES=(
+	"${FILESDIR}/${P}-030_manpages.patch"
+	"${FILESDIR}/${PN}-1.9.0-055_xstartup.patch"
+)
+
+src_prepare() {
+	# do not rely on the build system to install docs
+	sed -i 's:^\(install(.* DESTINATION ${DOC_DIR})\):#\1:' \
+		cmake/BuildPackages.cmake || die
+
+	cmake-utils_src_prepare
+
+	if use server ; then
+		cp -r "${WORKDIR}"/xorg-server-${XSERVER_VERSION}/. unix/xserver || die
+	fi
+
+	if use server ; then
+		BUILD_DIR="${WORKDIR}/${P}_build"
+		mkdir -p "${BUILD_DIR}/unix" || die
+		cp -r "${S}"/unix/xserver "${BUILD_DIR}"/unix || die
+		cp -r "${WORKDIR}"/xorg-server-${XSERVER_VERSION}/. "${BUILD_DIR}"/unix/xserver || die
+		cd "${BUILD_DIR}"/unix/xserver || die
+		eapply "${FILESDIR}"/xserver120.patch
+		eapply "${FILESDIR}"/xserver120-drmfourcc-header.patch
+		sed -i -e 's/"gl >= .*"/"gl"/' configure.ac || die
+		eautoreconf
+	fi
+}
+
+src_configure() {
+	if use arm || use hppa ; then
+		append-flags "-fPIC"
+	fi
+
+	local mycmakeargs=(
+		-DENABLE_GNUTLS=$(usex gnutls)
+		-DENABLE_NLS=$(usex nls)
+		-DENABLE_PAM=$(usex pam)
+		-DBUILD_JAVA=$(usex java)
+	)
+
+	cmake-utils_src_configure
+
+	if use server ; then
+		cd "${BUILD_DIR}"/unix/xserver || die
+		econf \
+			$(use_enable dri3) \
+			$(use_enable drm libdrm) \
+			$(use_enable opengl glx) \
+			$(use_enable xinerama) \
+			--disable-config-hal \
+			--disable-config-udev \
+			--disable-devel-docs \
+			--disable-dri \
+			--disable-glamor \
+			--disable-libunwind \
```

Comment:
You have arches keyworded that pretty much depend on unwind, could you make this optional?